bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -103,331 +97,68 @@ spec:
             memory: 300Mi
         terminationMessagePolicy: FallbackToLogsOnError
 
-      # nbdb: the northbound, or logical network object DB. In raft mode 
-      - name: nbdb
-        image: "{{.OvnImage}}"
+      - name: sb-ovsdb
+        image: "quay.io/mmahmoud/ovsdb-etcd:latest"
         command:
         - /bin/bash
         - -c
         - |
-          set -xe
-          if [[ -f /env/_master ]]; then
-            set -o allexport
-            source /env/_master
-            set +o allexport
-          fi
-
-          bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
-          # initialize variables
-          ovn_kubernetes_namespace=openshift-ovn-kubernetes
-          ovndb_ctl_ssl_opts="-p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt"
-          transport="ssl"
-          ovn_raft_conn_ip_url_suffix=""
-          if [[ "${K8S_NODE_IP}" == *":"* ]]; then
-            ovn_raft_conn_ip_url_suffix=":[::]"
-          fi
-          db="nb"
-          db_port="{{.OVN_NB_PORT}}"
-          ovn_db_file="/etc/ovn/ovn${db}_db.db"
-          # checks if a db pod is part of a current cluster
-          db_part_of_cluster() {
-            local pod=${1}
-            local db=${2}
-            local port=${3}
-            echo "Checking if ${pod} is part of cluster"
-            # TODO: change to use '--request-timeout=5s', if https://github.com/kubernetes/kubernetes/issues/49343 is fixed. 
-            init_ip=$(timeout 5 kubectl get pod -n ${ovn_kubernetes_namespace} ${pod} -o=jsonpath='{.status.podIP}')
-            if [[ $? != 0 ]]; then
-              echo "Unable to get ${pod} ip "
-              return 1
-            fi
-            echo "Found ${pod} ip: $init_ip"
-            init_ip=$(bracketify $init_ip)
-            target=$(ovn-${db}ctl --timeout=5 --db=${transport}:${init_ip}:${port} ${ovndb_ctl_ssl_opts} \
-                      --data=bare --no-headings --columns=target list connection)
-            if [[ "${target}" != "p${transport}:${port}${ovn_raft_conn_ip_url_suffix}" ]]; then
-              echo "Unable to check correct target ${target} "
-              return 1
-            fi
-            echo "${pod} is part of cluster"
-            return 0
-          }
-          # end of db_part_of_cluster
-          
-          # Checks if cluster has already been initialized.
-          # If not it returns false and sets init_ip to CLUSTER_INITIATOR_IP
+          set -x
           cluster_exists() {
-            local db=${1}
-            local port=${2}
-            # TODO: change to use '--request-timeout=5s', if https://github.com/kubernetes/kubernetes/issues/49343 is fixed. 
-            db_pods=$(timeout 5 kubectl get pod -n ${ovn_kubernetes_namespace} -o=jsonpath='{.items[*].metadata.name}' | egrep -o 'ovnkube-master-\w+' | grep -v "metrics")
+           ovsdb_etcd_members=${OVSDB_ETCD_MEMBERS:-"localhost:2479"}
```

Comment:
This configuration and the similar one for NB as well etcd server start arguments assume that there is a single etcd server and SB and NB are colocated with it in the same Pod. 

Will OCP run 3 master nodes with similar config on each of them ?
Should we start from a single master and after that change the configuration to support etcd cluster and several or dedicated ovsdb-etcd servers.